conan/internal/api/install/generators.py

```diff
@@ -151,6 +155,13 @@ def _receive_conf(conanfile):
             conanfile.conf.compose_conf(build_require.conf_info)
 
 
+def _receive_generators(conanfile):
```

Comment:
For ``conf_info`` we restricted it from ``tool_requires`` because practically all ``conf`` is focused on "build-time" things, and it didnt make much sense to have regular ``requires`` to define those things. It seems this is also the case, but lets think just in case there would be some other case that regular ``requires`` wanted to also inject generators.